awswrangler/catalog/_get.py

```diff
@@ -107,6 +107,7 @@ def get_table_types(
     database: str,
     table: str,
     catalog_id: str | None = None,
+    return_iceberg_current: bool = False,
```

Comment:
nit - `filter_iceberg_current` is clearer perhaps?

Reply:
Sure